src/opm/parser/eclipse/EclipseState/Schedule/Schedule.cpp

```diff
@@ -526,7 +526,7 @@ void Schedule::iterateScheduleSection(std::size_t load_start, std::size_t load_e
     }
 
     void Schedule::inspect_actionx_keyword(const ScheduleGrid& grid, const DeckKeyword& keyword){
-        static std::unordered_set<std::string> keyword_list = {"COMPDAT"};
+        static std::unordered_set<std::string> keyword_list = {"COMPDAT", "COMPSEGS"};
```

Comment:
The change is fine; but I would like you to rename the function. The purpose of this function is to load cell properties in advance - so I would suggest something like `prefetch_actionx_cell_properties` instead of `inspect_actionx_keyword`.